src/main/java/frc/robot/commands/CoralOuttake.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.SetPointConstants;
+import frc.robot.subsystems.end_effector.EndEffector;
+
+public class CoralOuttake extends Command{
```

Comment:
Our robot doesn't have the ability to outtake coral. One command I think would be helpful to have is `Purge`, which would run the funnel, coral, and algae rollers forward to push out any game pieces.